README.md

```diff
@@ -151,6 +144,10 @@ git submodule update --remote
 ### Updating patches
 
 ```sh
+# Assuming we are using an existing git clone of the ungoogled-chromium repo
+ln -s path/to/ungoogled-chromium  debian/ungoogled-upstream/ungoogled-chromium
+./debian/rules checkout-ungoogled-chromium
+
```

Comment:
These steps don't apply since we've switched to submodules.

Reply:
Removed